### PR TITLE
Fix/removePK

### PR DESCRIPTION
**Pull request by AnnRiera**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/blockcoders/harmony-marketplace-sdk/blob/main/CONTRIBUTING.md
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## What is the new behavior?
Removed Test PKs from Harmony SDK

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: .env.sample [hunk 1]
```diff
@@ -0,0 +1,4 @@
+TEST_PK_1='put your private key here'
+TEST_Pk_2='put your private key here'
+TEST_Pk_3='put your private key here'
```
Comment (0xslipk) [inline .env.sample:3]: this keys is not need right now. could you please remove it?

File: src/tests/constants.ts [hunk 2]
```diff
@@ -1,28 +1,69 @@
...
+import { PrivateKey } from '../private-key'
+import 'dotenv/config'
 
 export const HARMONY_TESTNET = 'https://api.s0.b.hmny.io/'
-export const TEST_ADDRESS_1 = '0x7c61c8f282aeaae3e0476d0533d21dc42c5aa146'.toLowerCase()
-export const TEST_ADDRESS_2 = '0xc57DE96E7ADd6D0fc6BdF51758Fd90b111f89b79'.toLowerCase()
-export const TEST_ADDRESS_3 = '0x264F47C0E2fb1B587F6fa270862F3CBc68460B75'.toLowerCase()
-export const TEST_ADDRESS_4 = '0xe8f887475466f6a232f428f038b9ada36dc3582d'.toLowerCase()
+
+const TEST_PK_1 = process.env.TEST_PK_1 ?? ''
```
Comment (0xslipk) [inline src/tests/constants.ts:9]: let's define all the env variables here, like:
```ts
const TEST_PK_1 = process.env.TEST_PK_1 ?? ''
const TEST_PK_2 = process.env.TEST_PK_2 ?? ''
const TEST_SEED = process.env.TEST_SEED ?? ''
...
// then if  validation.
```

File: .env.sample [hunk 3]
```diff
@@ -0,0 +1,4 @@
...
+TEST_SEED_1='pablo diego jose francisco de paula juan nepomuceno maria ruiz y picasso'
```
Comment (0xslipk) [inline .env.sample:4]: It's enough with `TEST_SEED` as name. It will be just one

File: src/tests/constants.ts [hunk 4]
```diff
@@ -1,28 +1,69 @@
...
+
+if (!TEST_PK_1) {
+  throw new Error('TEST_PK_1 must have a value')
```
Comment (0xslipk) [inline src/tests/constants.ts:12]: `...must be defined`

File: src/tests/constants.ts [hunk 5]
```diff
@@ -1,28 +1,69 @@
...
+}
+
+const TEST_SEED_1 = process.env.TEST_SEED_1 ?? ''
+
+if (!TEST_SEED_1) {
+  throw new Error('TEST_SEED_1 must have a value')
```
Comment (0xslipk) [inline src/tests/constants.ts:18]: `... must be defined`

File: src/tests/constants.ts [hunk 6]
```diff
@@ -1,28 +1,69 @@
...
-  privateKey: 'dc0ab9fc02cb694853c939037c986a792bdd779fe481c0767d7551ad45d45b39'.toLowerCase(),
+  address: PROVIDER_TEST_1.accounts[0].toLowerCase(),
+  privateKey: TEST_PK_1.toLowerCase(),
+  mnemonic: TEST_SEED_1.toLowerCase(),
+}
+
+const TEST_PK_2 = process.env.TEST_PK_2 ?? ''
+
+if (!TEST_PK_2) {
+  throw new Error('TEST_PK_2 must have a value')
```
Comment (0xslipk) [inline src/tests/constants.ts:36]: `... must be defined`

File: src/tests/constants.ts [hunk 7]
```diff
@@ -1,28 +1,69 @@
...
+  throw new Error('TEST_PK_1 must have a value')
+}
+
+const TEST_SEED_1 = process.env.TEST_SEED_1 ?? ''
+
+if (!TEST_SEED_1) {
+  throw new Error('TEST_SEED_1 must have a value')
+}
+
+export const PROVIDER_TEST_1: PrivateKey = new PrivateKey(
```
Comment (0xslipk) [inline src/tests/constants.ts:21]: let's rename this to `WALLET_PROVIDER_TEST_1`.

File: src/tests/constants.ts [hunk 8]
```diff
@@ -1,28 +1,69 @@
...
+  HarmonyShards.SHARD_0_TESTNET,
+  TEST_PK_1.toLowerCase(),
```
Comment (0xslipk) [inline src/tests/constants.ts:23]: The lower case in this case is not need it.

File: src/tests/constants.ts [hunk 9]
```diff
@@ -1,28 +1,69 @@
...
+  mnemonic: TEST_SEED_1.toLowerCase(),
+}
+
+const TEST_PK_2 = process.env.TEST_PK_2 ?? ''
+
+if (!TEST_PK_2) {
+  throw new Error('TEST_PK_2 must have a value')
 }
+
+export const PROVIDER_TEST_2: PrivateKey = new PrivateKey(
```
Comment (0xslipk) [inline src/tests/constants.ts:39]: let's rename this to `WALLET_PROVIDER_TEST_2`.

File: src/tests/constants.ts [hunk 10]
```diff
@@ -1,28 +1,69 @@
...
+  ChainID.HmyTestnet,
+)
 
 export const TEST_ACCOUNT_1 = {
-  address: TEST_ADDRESS_1,
-  privateKey: 'dc0ab9fc02cb694853c939037c986a792bdd779fe481c0767d7551ad45d45b39'.toLowerCase(),
+  address: PROVIDER_TEST_1.accounts[0].toLowerCase(),
+  privateKey: TEST_PK_1.toLowerCase(),
+  mnemonic: TEST_SEED_1.toLowerCase(),
+}
```
Comment (0xslipk) [inline src/tests/constants.ts:31]: I don't think we need this object anymore. Also the seed have to be a different key that we instance with `MnemonicKey`.

File: src/tests/constants.ts [hunk 11]
```diff
@@ -1,28 +1,69 @@
...
+export const PROVIDER_TEST_1: PrivateKey = new PrivateKey(
+  HarmonyShards.SHARD_0_TESTNET,
+  TEST_PK_1.toLowerCase(),
+  ChainID.HmyTestnet,
+)
 
 export const TEST_ACCOUNT_1 = {
-  address: TEST_ADDRESS_1,
-  privateKey: 'dc0ab9fc02cb694853c939037c986a792bdd779fe481c0767d7551ad45d45b39'.toLowerCase(),
+  address: PROVIDER_TEST_1.accounts[0].toLowerCase(),
```
Comment (0xslipk) [inline src/tests/constants.ts:28]: Just export an address after the PrivateKey instance, like:
```ts
...
export const ADDRESS_TEST_1 = WALLET_PROVIDER_TEST_1.accounts[0].toLowerCase()
export const ADDRESS_TEST_2 = WALLET_PROVIDER_TEST_2.accounts[0].toLowerCase()
// Wallet created with the seed.
export const ADDRESS_TEST_3 = WALLET_PROVIDER_TEST_3.accounts[0].toLowerCase() 
...
```

**Review by 0xslipk — CHANGES_REQUESTED** [review]

File: .env.sample (hunk 1 above)
Comment (AnnRiera) [reply .env.sample:3]: Sure thing!

File: .env.sample (hunk 3 above)
Comment (AnnRiera) [reply .env.sample:4]: Okay!

File: src/tests/constants.ts (hunk 4 above)
Comment (AnnRiera) [reply src/tests/constants.ts:12]: I'll fix it!

File: src/tests/constants.ts (hunk 5 above)
Comment (AnnRiera) [reply src/tests/constants.ts:18]: I'll fix it!

File: src/tests/constants.ts (hunk 6 above)
Comment (AnnRiera) [reply src/tests/constants.ts:36]: I'll fix it!

File: src/tests/constants.ts (hunk 7 above)
Comment (AnnRiera) [reply src/tests/constants.ts:21]: Copy!

File: src/tests/constants.ts (hunk 9 above)
Comment (AnnRiera) [reply src/tests/constants.ts:39]: Copy!

File: src/tests/constants.ts (hunk 10 above)
Comment (AnnRiera) [reply src/tests/constants.ts:31]: Copy!

File: src/tests/constants.ts (hunk 11 above)
Comment (AnnRiera) [reply src/tests/constants.ts:28]: Sure!

**Review by 0xslipk — APPROVED** [review]

**Pull request merged**